### ISSUE TITLE
issue with make when building

### ISSUE THREAD

**Issue by bensh** [ptoomey3/Keychain-Dumper#49]
Trying to get a binary that works on iOS 11/12 , so followed the build steps, but it fails on 'make'

macOS 10.14.6

    $ make
    xcrun --sdk iphoneos --find gcc` -Os -Wimplicit -isysroot `xcrun --sdk iphoneos --show-sdk-path` -arch armv7 -arch armv7s -arch arm64 -c main.m
    main.m:67:10: error: 'launchPath' is unavailable: not available on iOS
        task.launchPath = executablePath;
             ^
 ```/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Versions/C/Headers/NSTask.h:75:38: 
    note: 
          'launchPath' has been explicitly marked unavailable here
    @property (nullable, copy) NSString *launchPath 
    API_DEPRECATED_WITH_REPLACEMENT("executableU...
                                     ^
    main.m:71:11: error: 'launch' is unavailable: not available on iOS
        [task launch];
              ^```
 ```/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/Foundation.framework/Versions/C/Headers/NSTask.h:78:1: note: 
          'launch' has been explicitly marked unavailable here
    - (void)launch API_DEPRECATED_WITH_REPLACEMENT("launchAndReturnError:", macos(10.0, API_TO_B...
    ^
    2 errors generated.
    make: *** [main.o] Error 1```

**Comment by ptoomey3** [comment]
@mechanico - Can you comment on the above? It seems like the changes you had added were tested on 12.1 in https://github.com/ptoomey3/Keychain-Dumper/pull/38. Was support for this API removed after?

**Comment by mechanico** [comment]
Indeed I'm facing the same issue on my side. Unfortunately, I have no resources to chase down the issue now. I assume that an XCode update (compatibility with iOS 13) introduced this issue.

**Comment by mechanico** [comment]
As far as my investigation has come. This is the result:
- Apple has deprecated the create task feature we used to spawn openssl 

I tried messing around with my local files to re-enable the feature which did not work. So we could build a version of keychain_dumper without the openssl feature, or we could start building a version with openssl integrated. From previous projects I have seen some iOS apps with openssl integrated. This will increase the size of the binary but may give us the ability to circumvent this issue. From my  point of view, since i'm a mediocre developer, this task seems too big. But, since I have access to various jailbroken iOS versions I would be humbled to assist a more skilled developer with testing.
